.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
               echo "Merging $(cat merge.txt)";
               git remote add upstream https://github.com/scipy/scipy.git;
               git pull --ff-only upstream "refs/pull/$(cat merge.txt)/merge";
-              git fetch upstream master;
+              git fetch upstream main;
```

Comment:
of course, for backports PRs this wasn't correct before or after per gh-15425; I'm somewhat inclined to just adopt this quick fix for now and do the proper backport-sensitive fix later though